container/Dockerfile

```diff
@@ -1,26 +1,18 @@
 FROM centos:centos8
 
-# creature comforts
-ENV LANG en_US.UTF-8
-ENV CONTAINER docker 
-ENV USER sre-user
-ENV HOME /home/sre-user
-ENV TERM xterm
```

Comment:
out of curiosity, what's the reason for removing these env variables?

Reply:
The container isn't interactive, so they aren't needed by any process.

Reply:
ah, ok. i'd assumed they would be picked up by the shell once the user logs in.